aws/sagemaker/large-model-inference/sample-llm/tnx_speculative_decoding_deploy_llama2_70b.ipynb

```diff
@@ -0,0 +1,251 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "71a329f0",
+   "metadata": {},
+   "source": [
+    "# LLAMA 2 70B model speculative decoding deployment guide for Neuron instances\n",
+    "In this tutorial, you will use LMI container from DLC to SageMaker and run inference with it.\n",
+    "\n",
+    "Please make sure the following permission granted before running the notebook:\n",
+    "\n",
+    "- S3 bucket push access\n",
+    "- SageMaker access\n",
+    "\n",
+    "## Step 1: Let's bump up SageMaker and import stuff"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "67fa3208",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install sagemaker --upgrade  --quiet"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ec9ac353",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sagemaker\n",
+    "from sagemaker import Model, image_uris, serializers\n",
+    "\n",
+    "role = sagemaker.get_execution_role()  # execution role for the endpoint\n",
+    "sess = sagemaker.session.Session()  # sagemaker session for interacting with different AWS APIs\n",
+    "region = sess._region_name  # region name of the current SageMaker Studio environment\n",
+    "account_id = sess.account_id()  # account_id of the current SageMaker Studio environment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "81deac79",
+   "metadata": {},
+   "source": [
+    "## Step 2: Start preparing model artifacts\n",
+    "In LMI contianer, we expect some artifacts to help setting up the model\n",
+    "- serving.properties (required): Defines the model server settings\n",
+    "- model.py (optional): A python file to define the core inference logic\n",
+    "- requirements.txt (optional): Any additional pip wheel need to install"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b011bf5f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile serving.properties\n",
```

Comment:
Can we use this template: https://github.com/deepjavalibrary/djl-demo/blob/master/aws/sagemaker/large-model-inference/pysdk_template_with_LMI_solution.ipynb?